app/test/vehicles.js

```diff
@@ -0,0 +1,89 @@
+'use strict'
+
+let chai = require('chai')
+let should = chai.should()
+
+let vehicles = require('../vehicles')
+
+describe('When checking if vehicle has a outstanding recall', () => {
+  describe('and valid VIN and MARQUE is provided', () => {
+    it('and user provide upper case text and vehicle has a recall and "Recall Outstanding" message is provided.', () => {
```

Comment:
Think the reporting description won't be descriptive this way. Usually the standard way for units is to describe what we are testing in `describe` and outlining the expected behaviour in `it`.  So if those are unit tests we could perhaps
```
describe('vehicles' (...)
   describe('getRecall' (...)
      it('behaviour' (...)
```

What do you think?

Reply:
Discussed f2f